AUTHORS

```diff
@@ -214,6 +213,7 @@ Yommy <yomanda@gmail.com>
 Your Name <ibrahem.h.basyone@gmail.com>
 zackdreaver <zackdreaver@gmail.com>
 Zarbony <justonlinegames@web.de>
+Zopokx <zopokx@gmail.com>
```

Comment:
for rename commit authors need add rename string into ``.mailmap``
then generate authors list by script ``tools/authors.sh`` and replace first list of authors in AUTHORS file